.github/workflows/updatejson.yml

```diff
@@ -0,0 +1,27 @@
+name: Housekeeping
+on: 
+  push:
+  schedule:
+  #  Kick off job at midnight on the 1st and the 15th of every month
+  - cron: "0 0 1,15 * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps: 
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - id: run_py
+        run:  |
+        cd utils
+        python update_json_files
```

Comment:
@PhilipWright96 hmm. We also need to update our `goodreads.py` file to read the config(GOODREADS_PUBLIC_API_KEY, GOOGLE_SEARCH_RAPIDAPI_KEY etc) from environment variables right?

Reply:
Good shout - I'll get on this. Also I was wondering: is it safe to test this stuff on a fork? If I push up schedule changes to the master branch on a fork (as I will have to to test this stuff) , will those changes only live on my fork? Obviously a bit nervous about this!

Reply:
Yes. The changes would be present only in your fork. Forks can only update the original repo throughs pull request :)